### PR TITLE
Modularize managing packages

### PR DESCRIPTION
**Pull request by maximiliankolb**
#### What changes are you introducing?

make the content of the "Managing packages" chapter modular + minor changes to reflect the Foreman/Katello 3.11+ WebUI

#### Why are you introducing these changes? (Explanation, links to references, issues, etc.)

docs have to be modular to use it downstream.

#### Checklists

* [ ] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.12/Katello 4.14 (Satellite 6.16)
* [x] Foreman 3.11/Katello 4.13
* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9/6.10)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

**Comment by maximiliankolb** [comment]
Any idea if we want to keep "json" for syntax highlighting?
* https://docs.redhat.com/en/documentation/red_hat_satellite/6.15/html/managing_hosts/managing-packages_managing-hosts#Installing_Packages_on_a_Host_managing-hosts
* https://docs.theforeman.org/nightly/Managing_Hosts/index-katello.html#upgrading-packages-on-a-host_managing-hosts
* (*Managing packages* is not yet in orcharhino live docs)

Are there any technical reasons to keep it? Afaik this is the only example:

````
$ rg --no-filename "^\[" | rg -i json | rg -v "^\[id"
[options="nowrap", json, subs="+quotes,verbatim,attributes"]
````

File: guides/common/modules/proc_removing-packages-from-a-host.adoc [hunk 1]
```diff
@@ -5,10 +5,11 @@ You can remove packages from a host in the {ProjectWebUI}.
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
-. Click the name of the host you want to modify.
-. Click the *Content* tab, then click the *Packages* tab.
-. Click the vertical ellipsis for the package you want to remove and select *Remove*.
-You get a REX job notification once the remote execution job is complete.
+. Select a host.
+. On the *Content* tab, click the *Packages* tab.
```
Comment (apinnick) [inline guides/common/modules/proc_removing-packages-from-a-host.adoc:9]: This is outside the scope of your PR. I am auditing this guide and noticed that this assembly is very old. 

I logged in to Foreman and went to Hosts > All Hosts. Then I clicked a host to modify.

No Content tab, no Packages tab. I will open a bug.

Comment (apinnick) [reply guides/common/modules/proc_removing-packages-from-a-host.adoc:9]: Aha. Just found a RHEL host with a Content tab!

File: guides/common/modules/con_managing-packages.adoc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[id="managing-packages_{context}"]
+= Managing packages
+
+You can use {Project} to install, upgrade, and remove packages on hosts, as well as to enable or disable repositories on hosts.
```
Comment (apinnick) [inline guides/common/modules/con_managing-packages.adoc:4]:
```suggestion
You can use {Project} to install, upgrade, and remove packages and to enable or disable repositories on hosts.
```

File: guides/common/modules/proc_upgrading-packages-on-a-host.adoc [hunk 3]
```diff
@@ -5,13 +5,13 @@ You can upgrade packages on a host in bulk in the {ProjectWebUI}.
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
-. Click the name of the host you want to modify.
-. Click the *Content* tab, then click the *Packages* tab.
+. Select a host.
+. On the *Content* tab, click the *Packages* tab.
 . Select *Upgradable* from the *Status* list.
-. Select upgrade version from the dropdown menu in *Upgradable to* column where applicable.
+. In the *Upgradable to* column, select the package version that you want to upgreade to.
```
Comment (apinnick) [inline guides/common/modules/proc_upgrading-packages-on-a-host.adoc:11]:
```suggestion
. In the *Upgradable to* column, select the package version that you want to upgrade to.
```

File: guides/common/modules/con_managing-packages.adoc (hunk 2 above)
Comment (maximiliankolb) [reply guides/common/modules/con_managing-packages.adoc]: Applied.

File: guides/common/modules/proc_upgrading-packages-on-a-host.adoc (hunk 3 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_upgrading-packages-on-a-host.adoc]: Fixed.

File: guides/common/modules/proc_upgrading-packages-on-a-host.adoc [hunk 4]
```diff
@@ -5,13 +5,13 @@ You can upgrade packages on a host in bulk in the {ProjectWebUI}.
...
 . Select the packages you want to upgrade.
 . Click *Upgrade*.
-You get a REX job notification once the remote execution job is complete.
+You get a REX job notification after the remote execution job is complete.
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/modules/proc_upgrading-packages-on-a-host.adoc]: Reworded for all instances. I have moved the link to setup REX to the concept file.

**Review by maximiliankolb — COMMENTED** [review]
Rebased to "master" and applied all feedback. Ready for re-review.

**Comment by apinnick** [comment]
@maximiliankolb 

Where do you see this command? I don't see it in your PR.

> Any idea if we want to keep "json" for syntax highlighting?
> 
>     * https://docs.redhat.com/en/documentation/red_hat_satellite/6.15/html/managing_hosts/managing-packages_managing-hosts#Installing_Packages_on_a_Host_managing-hosts
> 
>     * https://docs.theforeman.org/nightly/Managing_Hosts/index-katello.html#upgrading-packages-on-a-host_managing-hosts
> 
>     * (_Managing packages_ is not yet in orcharhino live docs)
> 
> 
> Are there any technical reasons to keep it? Afaik this is the only example:
> 
> ```
> $ rg --no-filename "^\[" | rg -i json | rg -v "^\[id"
> [options="nowrap", json, subs="+quotes,verbatim,attributes"]
> ```

**Review by apinnick — APPROVED** [review]

**Comment by maximiliankolb** [comment]
Yes, I was trying to show that "json" syntax highlighting does not work; implying that we could drop it. For now, I'd keep it as is. I would create a follow-up issue/PR if I see a reason to change that.

**Pull request merged**